### PR TITLE
add 'show juttle' toggle to view juttle source

### LINKED ISSUE #59: Add ability to view juttle in outrigger
Mainly for demo purposes, I'd like to have a way to show the code of the loaded juttle program when I'm running it in outrigger. A "view juttle" link that I click to open an area with code above the visualizations would be fine (with "hide juttle" to disappear it again). The minimal solution would open the juttle code in a new browser tab.

### PR DESCRIPTION
**Pull request by go-oleg**
Add a basic show/hide juttle toggle link
- starts off hidden
- expands to max-height of 200px and then scrolls

Resolves #59 

image

image

let me know if you want any styling changes

@mnibecker @davidvgalbraith 

cc @dmehra

### CODE REVIEW

File: src/apps/run/main.js [hunk 1]
```diff
@@ -12,14 +17,17 @@ let outriggerHost = window.location.host;
 let client = new Juttle(outriggerHost);
 let view = new client.View(document.getElementById("juttle-view-layout"));
 let inputs = new client.Input(document.getElementById("juttle-input-groups"));
+let juttleSoureEl = document.getElementById("juttle-source");
```
Comment (davidvgalbraith) [inline src/apps/run/main.js]: `juttleSourceEl`, I think you meant. (there's a `c` in there)

**Comment by davidvgalbraith** [comment]
Is this definitely part of outrigger? Seems like it would be at home in the Juttle client library.

File: src/apps/run/main.js (hunk 1 above)
Comment (go-oleg) [inline src/apps/run/main.js]: ah oops, fixed.

**Comment by go-oleg** [comment]
I don't think it really belongs in juttle client library. My understanding is that juttle-client-library is meant to be used in other applications for running juttle and showing juttle inputs and visualizations, not the actual juttle code. @mnibecker, thoughts?

**Comment by go-oleg** [comment]
@mnibecker bump

**Comment by mnibecker** [comment]
Siding with Oleg here. I think code should not be in `juttle-client-library`.

**Comment by mnibecker** [comment]
+1

**Comment by dmehra** [comment]
For demo purposes, I'm requesting to open the code window to program size, so I don't have to scroll it.

**Comment by go-oleg** [comment]
@dmehra, done

**Comment by go-oleg** [comment]
@davidvgalbraith, further thoughts/are you cool with this?

**Comment by davidvgalbraith** [comment]
+1

**Pull request merged**